viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -140,6 +140,13 @@
           Unlock
         </button> <!-- /unlock button-->
 
+        <!-- export button-->
+        <button class="btn btn-default btn-sm ml-1"
+          @click.stop.prevent="exportPng()">
```

Comment:
change exportPng() to exportPng

Reply:
I have done so, see 5349cca

Reply:
For what it's worth, I was just copying the style of the unlock button right above this one. It seems several of the functions in that file had the ()

Reply:
Good find! I will update all of those (I've learned a lot about Vue since creating this page).